test/testing_utils.dart

```diff
@@ -0,0 +1,224 @@
+import 'angular_utils.dart';
+import 'dart:math';
+import 'package:webdriver/io.dart';
+import 'package:random_string/random_string.dart' as random;
+import 'dart:async';
+import 'dart:io';
+
+void startSelenium() {
+  print("Starting Selenium Chrome Driver");
+  Process.run(
+      'java.exe -Ddriver.chrome.driver=chromedriver.exe -jar '
+      'selenium-server-standalone-3.6.0.jar',
+      []);
+}
+
+Future<WebDriver> setupWebDriver(int width, int height) async {
+  var options = {'browserName': 'chrome'};
+  WebDriver driver = await createDriver(desired: options);
+  driver.logs
+      .get('browser')
+      .where((LogEntry l) => l.level == 'SEVERE')
+      .listen((LogEntry l) => print(l));
+  await (await driver.window).setSize(new Rectangle<int>(0, 0, width, height));
+  return driver;
+}
+
+Future navigateToPage(WebDriver driver, String slug) async {
+  await driver.get("https://www.gyerunkanyukam.hu/" + slug);
+}
+
+Future<String> signUpNewRandomUser(WebDriver driver, String pw) async {
```

Comment:
This should be moved to the other test groups (it is not a generic purpose utility function)